tests/_fixtures/marathon-1-task/groups-sdk-services.json

```diff
@@ -66,54 +46,39 @@
           "tasksRunning": 1,
           "tasksHealthy": 0,
           "tasksUnhealthy": 0,
-          "deployments": [
-
-          ]
+          "deployments": []
         },
         {
           "id": "/services/sdk-sleep-with-image",
           "cmd": "sleep 3000",
           "args": null,
           "user": null,
-          "env": {
-
-          },
+          "env": {},
           "instances": 1,
           "cpus": 0.1,
           "mem": 16,
           "disk": 0,
           "executor": "",
-          "constraints": [
-
-          ],
-          "uris": [
-
-          ],
-          "storeUrls": [
-
-          ],
-          "ports": [
-            10000
-          ],
+          "constraints": [],
+          "uris": [],
+          "storeUrls": [],
+          "ports": [10000],
           "requirePorts": false,
           "backoffSeconds": 1,
           "backoffFactor": 1.15,
           "maxLaunchDelaySeconds": 3600,
           "container": null,
-          "healthChecks": [
-
-          ],
-          "dependencies": [
-
-          ],
+          "healthChecks": [],
+          "dependencies": [],
           "upgradeStrategy": {
             "minimumHealthCapacity": 1,
             "maximumOverCapacity": 1
           },
           "labels": {
             "DCOS_COMMONS_API_VERSION": "v1",
             "DCOS_PACKAGE_NAME": "sdk-sleep",
-            "DCOS_PACKAGE_FRAMEWORK_NAME": "/services/sdk-sleep"
+            "DCOS_PACKAGE_FRAMEWORK_NAME": "/services/sdk-sleep",
+            "DCOS_SERVICE_WEB_PATH": "/test"
```

Comment:
Keeping the important part ofcourse. VSCode has an option to save without reformating